inorbit_republisher/config/example.yaml

```diff
@@ -54,6 +54,16 @@ republishers:
       topic: "/inorbit/custom_data/0"
       key: "navsat"
 
+# Añadido para los tópicos input y output
+- topic: "/input_topic"
```

Comment:
I wouldn't use this file for testing purposes, as it's the example used on the main README. Instead, I'd create a separate configuration file e.g. `unittest_config.yaml` under the test folder, and only include this section.

Reply:
Solved